projects/netgrif-components-core/src/lib/utility/tests/mocks/mock-authentication.service.ts

```diff
@@ -4,7 +4,9 @@ import {Observable, of} from 'rxjs';
 import {User} from '../../../user/models/user';
 import {Injectable} from '@angular/core';
 
-@Injectable()
+@Injectable({
+    providedIn: 'root'
```

Comment:
the Mock service should not be provided in `root`, because it then becomes a part of the providers of the library/module, which is not desired